source/js/foundation/template-js-handler/publication-summary-links.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Checking whether the page has more than 5 items in the summary link section, if so, display them in a two column format.
+ * If < 5 links, display them in one column
+ */
+export default () => {
+  const summaryArticleSection = document.querySelector(".article-links");
+  const countOfArticleLinks = document.querySelectorAll(".article-summary-link").length;
+
+  if (countOfArticleLinks > 5) {
+    summaryArticleSection.classList.add("article-links__two-col");
```

Comment:
We don't typically work with BEM in the foundation website. `article-links-two-col` or `article-links-grid` or something might be a better name.